packages/paima-sdk/paima-mw-core/src/state.ts

```diff
@@ -17,17 +17,18 @@ let gameName: string = '';
 let backendUri: URI = ENV.BACKEND_URI;
 const batcherUri: URI = ENV.BATCHER_URI;
 
-const chainUri: URI = ENV.CHAIN_URI;
-const chainExplorerUri: URI = ENV.CHAIN_EXPLORER_URI;
-const chainId: number = ENV.CHAIN_ID;
-const chainName: string = ENV.CHAIN_NAME;
-const chainCurrencyName: string = ENV.CHAIN_CURRENCY_NAME;
-const chainCurrencySymbol: string = ENV.CHAIN_CURRENCY_SYMBOL;
-const chainCurrencyDecimals: number = ENV.CHAIN_CURRENCY_DECIMALS;
+const [chainName, config] = await GlobalConfig.mainEvmConfig();
```

Comment:
FYI this will cause issues when called from frontends because `readFile` is not available from web browsers which is the same problem we had with `SECURITY_NAMESPACE` (now that I think of it, maybe `SECURITY_NAMESPACE` should be merged into the same file as this config)

Reply:
Right, I was concerned about that, but I wasn't sure what's included in browsers and what not. But what's the fix for that? Should I keep the ENV variables for this case? They could be filled to the main network settings after all.

Reply:
as we discussed in the call, it may not be required for most frontend if it fallsback to the .env file the basic stuff which is not great behavior, but not bad as a temporary solution. We just have to make sure that `mainEvmConfig` doesn't crash the app by trying to call `readFile` when running from the frontend and properly document this temporary behavior

Reply:
I reverted this file to use the env file settings. I'm not sure if the change would also be needed somewhere else.